src/bin/bitmaskd.rs

```diff
@@ -46,7 +46,7 @@ async fn import(Json(asset): Json<AssetRequest>) -> Result<impl IntoResponse, Ap
 
 #[axum_macros::debug_handler]
 async fn transfer(Json(transfer): Json<TransferRequest>) -> Result<impl IntoResponse, AppError> {
-    let (consignment, psbt, disclosure, _, previous_utxo, _) = transfer_assets(
```

Comment:
Please use destructuring assignment, similar to this:
https://github.com/diba-io/bitmask-core/blob/42dc3fad86c8a2b0a8d290950744fb10c07742b0/src/lib.rs#L570-L575